test/models/incomeSourceDetails/incomeSourceIds/IncomeSourceIdSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails.incomeSourceIds
+
+import models.incomeSourceDetails.incomeSourceIds.IncomeSourceId
+import models.incomeSourceDetails.incomeSourceIds.IncomeSourceId.mkIncomeSourceId
+import testConstants.BaseTestConstants.testSelfEmploymentId
+import testUtils.UnitSpec
+
+class IncomeSourceIdSpec extends UnitSpec {
+
+  "IncomeSourceId object" should {
+    "return the overridden toString of the incomeSourceId" when {
+      "on an incomeSourceId object" in {
+        val incomeSourceId: IncomeSourceId = mkIncomeSourceId(testSelfEmploymentId)
+        val incomeSourceIdValue = incomeSourceId.value
+        val incomeSourceIdToString: String = s"IncomeSourceId: $incomeSourceIdValue"
```

Comment:
.toString is a helper method, no need to test , you can actually compare with value created out of mkFromQueryString

Reply:
Since this is the only test in the IncomeSourceIdSpec should I just remove the spec altogether, or is there something else I can test